File: c/03-estrutura-repetitiva/fatorial.c
/*Problema "fatorial" (adaptado de URI 1153)
Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o
fatorial de N.*/

#include <stdio.h>
int main () {
    int n, fatorial = 1;
    printf("Digite o valor de N: ");
    scanf("%d", &n);

    if (n > 15) {
        n = 15;
    }

    for(int i = 1; i <= n; i++) {
        fatorial = i * fatorial;
    }

    printf("FATORIAL = %d\n", fatorial);

}

File: java/03-estruturas-repetitivas/Fatorial.java
/*Problema "fatorial" (adaptado de URI 1153)
Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o
fatorial de N.*/

import java.util.Locale;
import java.util.Scanner;

public class Fatorial {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, fatorial = 1;
		
		System.out.print("Digite o valor de N: ");
		n = sc.nextInt();
		
		for (int i = 1; i <= n; i ++) {
			fatorial = i * fatorial;
		}
		
		System.out.println("FATORIAL = " + fatorial);
		
		
		sc.close();
	}

}
